elasticjob-lite/elasticjob-lite-spring/pom.xml

```diff
@@ -34,7 +34,7 @@
 
     <properties>
         <springboot.version>2.3.1.RELEASE</springboot.version>
-        <springframework.version>[3.1.0.RELEASE,)</springframework.version>
```

Comment:
The class `org.springframework.util.ConcurrentLruCache` is added since `5.3.0`.
If we don't specify the upper bound of Spring version, a part of Spring modules will be upgraded to `5.3.0`.
But some modules are still the lower version.

Ref: https://github.com/spring-projects/spring-framework/blob/master/spring-core/src/main/java/org/springframework/util/ConcurrentLruCache.java